parsers/ENTSOE.py

```diff
@@ -779,21 +756,53 @@ def parse_production(
                 len(timeseries.find_all("inBiddingZone_Domain.mRID".lower())) > 0
             )
             dt = datetime_from_position(datetime_start, position, resolution)
-            production, storage = create_production_storage(
-                fuel_code, quantity if is_production else -quantity, logger, zoneKey
+            list_of_raw_data.append(
```

Comment:
Would be nice to split this new functionality into it's own function and clearly document what's going on - it's pretty hard to read and understand right now in my opinion :)

Reply:
Will try to do so as soon as possible :)

Reply:
Don't think I can add much more comments than what I just did without it getting less understandable but let me know if you think anything is unclear.